shell/config/table-headers.js

```diff
@@ -256,6 +256,21 @@ export const DESCRIPTION = {
   width:    300,
 };
 
+export const NS_SNAPSHOT_QUOTA = {
+  name:          'NamespaceSnapshotQuota',
+  labelKey:      'harvester.tableHeaders.totalSnapshotQuota',
```

Comment:
This has to be: 'tableHeaders.totalSnapshotQuota' to get the label from shell translations yaml.

Reply:
I was using 'tableHeaders.totalSnapshotQuota' but found current released rancher 2.8 and 2.9 don't have such header. It will causes user see unexpected translation key if imported harvester.